client/blocks/reader-post-card/photo.jsx

```diff
@@ -102,17 +115,26 @@ class PostPhoto extends React.Component {
 			: {};
 
 		return (
+			<div style={ divStyle } >
+				<a className={ classes } href={ post.URL } style={ featuredImageStyle } onClick={ this.handleClick }>
+					<div ref={ this.handleWidthDivLoaded } style={ { width: '100%' } }></div>
+				</a>
+				<AutoDirection>
+					<h1 className="reader-post-card__title">
+						<a className="reader-post-card__title-link" href={ post.URL } onClick={ this.props.onClick }>{ linkTitle }</a>
```

Comment:
I didn't see the click handler attached to the title in #10503 but I checked with @fraying to make sure that clicking the photo title should open the full post view.